boards/arm/sam4l_ek/sam4l_ek.dts

```diff
@@ -61,3 +61,20 @@
 	cs-gpios = <&gpioa 2 GPIO_ACTIVE_LOW>,
 		   <&gpioc 0 GPIO_ACTIVE_LOW>;
 };
+
+&twim0 {
+	status = "okay";
+
+	std-clk-slew-lim = <0>;
```

Comment:
Copying this sequence of property defaults as a block into the top-level documentation for the binding would address my concern above.